tests/test_mock_controller.py

```diff
@@ -85,10 +88,10 @@ async def test_moveAz(self):
             amcs_status["status"], Dome.LlcStatus.MOVING.value,
         )
         self.assertGreaterEqual(
-            amcs_status["positionActual"], 0.5,
+            amcs_status["positionActual"], 0.5 * _DEGREES_TO_RADIANS,
```

Comment:
consider import numpy as np, and then

```python
amcs_status["positionActual"], np.radians(0.5),
```

Reply:
I resolved to using math.radians and math.degrees in almost all cases, except where a list was involved where I am using numpy instead.